src/quicer_nif.erl

```diff
@@ -424,3 +431,20 @@ priv_dir() ->
         Dir ->
             Dir
     end.
+
+%% @doc Get the load balancing server id from the given device name. ipv4 only.
+-spec lb_server_id(ipv4, string()) -> non_neg_integer().
+lb_server_id(ipv4, DevName) ->
+    try
+        {ok, IfList} = inet:getifaddrs(),
+        %% @NOTE Be aware of the order of the bytes in the address
+        lists:foldr(
+            fun(I, V) -> V bsl 8 bor I end,
+            0,
+            tuple_to_list(proplists:get_value(addr, proplists:get_value(DevName, IfList)))
+        )
+    catch
+        _:E ->
+            logger:error("Failed to set lb mode from ~s, fallback to disabled: ~p", [DevName, E]),
```

Comment:
Nit. I'd vote for more transparent error handling here, instead of having `{function_clause, ...}` errors with huge stacktraces in logs just because of `proplists:get_value(addr, undefined)` call.

Reply:
This is only called during module load, stacktrace won't be huge, and most likely shoudn't fail if config is correct. 

not only because of `proplists:get_value`, `inet:getifaddrs()` may also fail, not much we could do to handle the error here ,logging should be enough.

Reply:
Since it's during module load, maybe if it were to fail fast, the user would notice earlier that something is wrong/unexpected.